src/app/directives/documentation.tpl.html

```diff
@@ -186,6 +186,33 @@ <h4 class="raml-console-resource-body-heading">
             <pre class="raml-console-resource-pre"><code class="raml-console-hljs" hljs source="getBeatifiedExample(responseInfo[code][responseInfo[code].currentType].example)"></code></pre>
           </div>
 
+          <div ng-if="responseInfo[code][responseInfo[code].currentType].examples">
+            <span>Examples:</span>
+            <div ng-if="isArray(responseInfo[code][responseInfo[code].currentType].examples)">
```

Comment:
Let's convert it to an array at controller level and assume we always deal with array at template level. It's going to simplify it a lot.